src/app/cart/_components/coupons.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { cn } from "@/lib/utils";
+import { Coupon, CouponType, useCoupons } from "@/services/coupon";
+import { usePageContext } from "../_context";
+
+export const Coupons = () => {
+  const { data: coupons } = useCoupons();
+
+  return (
+    <div className="mt-16 flex flex-col">
+      <h2 className="mb-8 text-2xl font-bold">쿠폰</h2>
+      <ul className="flex flex-col gap-4">
+        {coupons.map((coupon) => (
+          <CouponListItem key={coupon.id} coupon={coupon} />
```

Comment:
요렇게 표현되어도 충분할 것 같네요.
```suggestion
          <CouponItem key={coupon.id} coupon={coupon} />
```